mlrun/api/api/endpoints/marketplace.py

```diff
@@ -67,6 +67,9 @@ async def create_source(
     response_model=List[IndexedMarketplaceSource],
 )
 async def list_sources(
+    item_name: Optional[str] = Query(None),
```

Comment:
```suggestion
    item_name: Optional[str] = Query(None, alias="item-name"),
```
We only use dashes in REST parameter names.